ResultsComparer build is broken

Getting the following errors since April 07, 2020:

```
2020-04-07T11:49:12.9453125Z Program.cs(14,35): error CS0234: The type or namespace name 'StatisticalTesting' does not exist in the namespace 'BenchmarkDotNet.Mathematics' (are you missing an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9464548Z Program.cs(64,174): error CS0246: The type or namespace name 'Threshold' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9466593Z Program.cs(64,199): error CS0246: The type or namespace name 'Threshold' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9468606Z Program.cs(64,92): error CS0246: The type or namespace name 'EquivalenceTestConclusion' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9470542Z Program.cs(84,98): error CS0246: The type or namespace name 'EquivalenceTestConclusion' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9472376Z Program.cs(114,96): error CS0246: The type or namespace name 'EquivalenceTestConclusion' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9474267Z Program.cs(114,145): error CS0246: The type or namespace name 'EquivalenceTestConclusion' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9476146Z Program.cs(170,97): error CS0246: The type or namespace name 'EquivalenceTestConclusion' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9478017Z Program.cs(190,97): error CS0246: The type or namespace name 'EquivalenceTestConclusion' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9480011Z Program.cs(265,96): error CS0246: The type or namespace name 'EquivalenceTestConclusion' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
2020-04-07T11:49:12.9482402Z Program.cs(267,40): error CS0246: The type or namespace name 'EquivalenceTestConclusion' could not be found (are you missing a using directive or an assembly reference?) [/home/ubuntu/agent/_work/1/s/work/performance/src/tools/ResultsComparer/ResultsComparer.csproj]
```

The last successful build was from day before that.

--- a/src/tools/ResultsComparer/Program.cs
+++ b/src/tools/ResultsComparer/Program.cs
@@ -10,8 +10,9 @@ using System.Linq;
 using System.Text.RegularExpressions;
 using System.Threading;
 using System.Xml;
-using BenchmarkDotNet.Mathematics;
-using BenchmarkDotNet.Mathematics.StatisticalTesting;
+using Perfolizer.Mathematics.Multimodality;
+using Perfolizer.Mathematics.SignificanceTesting;
+using Perfolizer.Mathematics.Thresholds;
 using CommandLine;
 using DataTransferContracts;
 using MarkdownLog;
@@ -251,7 +252,7 @@ namespace ResultsComparer
             if (benchmark.Statistics.N < 12) // not enough data to tell
                 return null;
 
-            double mValue = MathHelper.CalculateMValue(new BenchmarkDotNet.Mathematics.Statistics(benchmark.GetOriginalValues()));
+            double mValue = MValueCalculator.Calculate(benchmark.GetOriginalValues());
             if (mValue > 4.2)
                 return "multimodal";
             else if (mValue > 3.2)

--- a/src/tools/ResultsComparer/ResultsComparer.csproj
+++ b/src/tools/ResultsComparer/ResultsComparer.csproj
@@ -10,5 +10,6 @@
     <PackageReference Include="MarkdownLog.NS20" Version="0.10.1" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
     <PackageReference Include="BenchmarkDotNet" Version="0.12.1" />
+    <PackageReference Include="Perfolizer" Version="0.2.1" />
   </ItemGroup>
 </Project>

